bigchaindb/util.py

```diff
@@ -109,8 +110,7 @@ def serialize(data):
         str: JSON formatted string
 
     """
-    return json.dumps(data, skipkeys=False, ensure_ascii=False,
-                      separators=(',', ':'), sort_keys=True)
+    return rapidjson.dumps(data, skipkeys=False, ensure_ascii=False, sort_keys=True)
```

Comment:
At least the python wrapper does not provide that argument but it uses `(',', ':')` by default